common/src/main/java/com/microsoft/identity/common/internal/broker/PackageHelper.java

```diff
@@ -140,13 +140,6 @@ public boolean isPackageInstalledAndEnabled(final Context context, final String
     public static String getBrokerRedirectUrl(final String packageName, final String signatureDigest) {
         if (!StringExtensions.isNullOrBlank(packageName)
                 && !StringExtensions.isNullOrBlank(signatureDigest)) {
```

Comment:
The thing to watch for here is dual installations of CompanyPortal/Authenticator.  If the new Authenticator is trying to talk to a broker that's actually CP, what happens in that case (it will return the old uri)?    Additionally, if an old authenticator tries to connect to a CP that's based on this new code, what happens?

Reply:
Part of address this concern is covered in this MSAL PR  https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1517